src/libs/actions/Report.js

```diff
@@ -697,37 +697,63 @@ function addComment(reportID, text) {
  * Gets the latest page of report actions and updates the last read message
  *
  * @param {String} reportID
+ * @param {Array} participantList
+ * @param {Object} newReportObject
  */
-function openReport(reportID) {
+function openReport(reportID, participantList = [], newReportObject = {}) {
+    const onyxData = {
+        optimisticData: [{
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {
+                isLoadingReportActions: true,
+                lastVisitedTimestamp: Date.now(),
+                lastReadSequenceNumber: getMaxSequenceNumber(reportID),
+            },
+        }],
+        successData: [{
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {
+                isLoadingReportActions: false,
+                pendingFields: {
+                    createChat: null,
+                },
+                errorFields: {
+                    createChat: null,
+                },
+                isOptimisticReport: false,
+            },
+        }],
```

Comment:
Do we have a way to clear the failed red brick road error here? Like an error is set via PHP but how do you clear the failed created chat?

Reply:
If the chat creation fails, we make you delete the report from the error so it clears everything from onyx